src/app/Main.java

```diff
@@ -126,15 +135,39 @@ public static User userExists(String name, String lastname, ArrayList<User> user
         }
         return null;
     }
-    public static Book bookExists(String name,String author,int year,ArrayList<Book> books){
+
+
+    public static Book bookExists(String name, String author, int year, ArrayList<Book> books) {
         for (Book book : books) {
-            if (book.equals(name, author, year)){
+            if (book.equals(name, author, year)) {
                 return book;
             }
         }
         System.out.println("Book not found");
         return null;
     }
+
+    public static ArrayList<Reserva> searchReservation(String name, String lastname, ArrayList<Reserva> reservas) {
+        ArrayList<Reserva> userReservation = new ArrayList<>();
+        for (Reserva reserva : reservas) {
+            if (reserva.getUser().getName().equals(name) && reserva.getUser().getLastname().equals(lastname)) {
+                userReservation.add(reserva);
+            }
+        }
+        return userReservation;
+
+    }
+
+
+    public static Reserva reservationExist(Book book, User user, ArrayList<Reserva> reservas) {
```

Comment:
Where is this being used?

Reply:
This is trash code, I must remove it before merge to master.